src/components/idleUsers/card/card.module.scss

```diff
@@ -25,9 +25,13 @@ $theme-pink: #e30464;
   }
 
   .name {
-    margin-top: 3%;
-    margin-bottom: 0;
+    display: flex;
     font-size: 1rem;
     font-weight: 700;
+    width: 160px;
```

Comment:
Can we give width in `rem` instead of `px`? As rem is preferrable when value is greater than 20px.

Reply:
updated it